src/components/elements/table/header.js

```diff
@@ -1,21 +1,53 @@
+import { Component } from 'react'
 import cx from 'classnames'
 
-export default ({ api, property }) => {
-  if (!property.columns) {
-    return <thead/>
+export default class Header extends Component {
+  constructor(props) {
+    super()
+
+    this.state = {
+      sortedColumn: null,
+      direction: null,
```

Comment:
Maybe this property should be named `descending` instead considering how it is used.